src/components/TextInput/TextInput.tsx

```diff
@@ -89,6 +94,7 @@ export const TextInput: FC<TextInputProps> = ({
     autocomplete = false,
     dotted = false,
     value = "",
+    copyable,
```

Comment:
Probably it makes sense to define false as a default value

Reply:
done